src/support/mtx_rw.c

```diff
@@ -260,12 +262,16 @@ stall:			__wt_cond_wait(session,
 	}
 	if (set_stats) {
 		time_stop = __wt_clock(session);
+		time_diff = WT_CLOCKDIFF_US(time_stop, time_start);
 		if (F_ISSET(session, WT_SESSION_INTERNAL))
 			stats[session->stat_bucket][l->stat_int_usecs_off] +=
-			    (int64_t)WT_CLOCKDIFF_US(time_stop, time_start);
-		else
+			    (int64_t)time_diff;
+		else {
 			stats[session->stat_bucket][l->stat_app_usecs_off] +=
```

Comment:
same as above, have we decided not to track this stat for the internal sessions?